apps/axelarscan-amplifier-apis/src/configs/feesAndPrices.ts

```diff
@@ -24,6 +24,14 @@ const feesAndPrices = {
       approvalCost: 250000,
       updated: "2024-09-05",
     },
+    {
+      name: "test-hedera",
```

Comment:
it's just `hedera` in this case: https://github.com/axelarnetwork/axelar-contract-deployments/blob/main/axelar-chains-config/info/testnet.json#L2411